### PR TITLE
Fix logouts and refreshes

### PR DESCRIPTION
**Pull request by chrroberts-pure**
While experimenting with some go auto tracing - it came to my attention that our logouts are not formatted correctly and are failing leaving sessions open.

This PR fixes this issue by setting the correct base for this call `/api/logout` and then re-sets `/api/<version>/logout`

BEFORE
````
==============================================================================
2024/10/22 23:18:53.941631 DEBUG RESTY 
==============================================================================
~~~ REQUEST ~~~
POST  /api/2.15/logout  HTTP/1.1
HOST   : fb06
HEADERS:
        Accept: application/json
        Content-Type: application/json
        User-Agent: Pure_FB_OpenMetrics_exporter/1.0.13 (PostmanRuntime/7.41.2)
        X-Auth-Token: 989fe388-abcd-efgh-8746-32d4a2cf691d
BODY   :
***** NO CONTENT *****
------------------------------------------------------------------------------
~~~ RESPONSE ~~~
STATUS       : 400 Bad Request
PROTO        : HTTP/2.0
RECEIVED AT  : 2024-10-22T23:18:53.941601-05:00
TIME DURATION: 60.205702ms
HEADERS      :
        Cache-Control: no-cache, no-store, max-age=0, must-revalidate
        Content-Length: 0
        Content-Security-Policy: frame-ancestors 'none'
        Date: Wed, 23 Oct 2024 04:18:54 GMT
        Expires: 0
        Pragma: no-cache
        Server: nginx
        Strict-Transport-Security: max-age=31536000; includeSubDomains;
        Vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
        X-Content-Type-Options: nosniff
        X-Frame-Options: DENY
        X-Xss-Protection: 1; mode=block
BODY         :
***** NO CONTENT *****
==============================================================================
````

AFTER
````
==============================================================================
2024/10/22 23:22:09.120715 DEBUG RESTY 
==============================================================================
~~~ REQUEST ~~~
POST  /api/logout  HTTP/1.1
HOST   : fb06
HEADERS:
        Accept: application/json
        Content-Type: application/json
        User-Agent: Pure_FB_OpenMetrics_exporter/1.0.13 (PostmanRuntime/7.41.2)
        X-Auth-Token: 817b6df5-abcd-efgh-88e7-0add9156b8f2
BODY   :
***** NO CONTENT *****
------------------------------------------------------------------------------
~~~ RESPONSE ~~~
STATUS       : 200 OK
PROTO        : HTTP/2.0
RECEIVED AT  : 2024-10-22T23:22:09.12066-05:00
TIME DURATION: 59.40881ms
HEADERS      :
        Cache-Control: no-cache, no-store, max-age=0, must-revalidate
        Content-Length: 0
        Content-Security-Policy: frame-ancestors 'none'
        Date: Wed, 23 Oct 2024 04:22:09 GMT
        Expires: 0
        Pragma: no-cache
        Server: nginx
        Strict-Transport-Security: max-age=31536000; includeSubDomains;
        Vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
        X-Auth-Token:
        X-Content-Type-Options: nosniff
        X-Frame-Options: DENY
        X-Xss-Protection: 1; mode=block
BODY         :
***** NO CONTENT *****
==============================================================================
````

### CODE REVIEW

**Comment by chrroberts-pure** [comment]
I want to wait 7 days for testing on this, as previous changes to the client has caused issues in the past

**Review by sdodsley — APPROVED** [review]
Looks good

**Comment by chrroberts-pure** [comment]
closes #91

**Comment by chrroberts-pure** [comment]
tested over a week, and looks good and approved for v1.1.0

File: Makefile [hunk 1]
```diff
@@ -3,7 +3,7 @@ GOTEST=$(GOCMD) test
 GOVET=$(GOCMD) vet
 BINARY_NAME=pure-fb-om-exporter
 MODULE_NAME=purestorage/fb-openmetrics-exporter
-VERSION?=1.0.13
+VERSION?=1.0.14
```
Comment (chrroberts-pure) [inline Makefile:6]:
```suggestion
VERSION?=1.0.13
```

File: build/docker/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 FROM golang:alpine as build
-ARG VERSION=1.0.13
+ARG VERSION=1.0.14
```
Comment (chrroberts-pure) [inline build/docker/Dockerfile:2]:
```suggestion
ARG VERSION=1.0.13
```

**Pull request merged**